### DISCUSSION TITLE
GUIDE: sound notification when flyscrape is done

### DISCUSSION THREAD

**Discussion by dynabler** [philippta/flyscrape#51] [Show and tell]
If you like to get a sound notification when Flyscrape is done, you can use the following commands [source](https://stackoverflow.com/questions/1143386/in-a-bash-script-command-how-can-i-make-a-pc-beep-noise-or-play-a-sound-file). Feel free to comment your solution.

### Windows
Make sure system sounds are turned on

`$ flyscrape run script.js && echo -en "\007"`

### OSX
`$ flyscrape run script.js && say "flyscrape is done"`

### Linux
`$ flyscrape run script.js && spd-say 'flyscrape is done' `

**Comment by philippta** [comment]
That's really cool! Thanks for sharing. 🥳
